source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst

```diff
@@ -117,6 +117,12 @@ Initial Topology Description
 
 The first ``TopologyDescriptionChangedEvent`` to be emitted from a monitored Topology MUST set its ``previousDescription`` property to be a ``TopologyDescription`` object in the "unknown" state.
 
+Closing Topology Description
+----------------------------
+
+When a ``Topology`` object or equivalent is being shut-down or closed, the driver SHOULD change the
+``TopologyDescription`` to an "unknown" state.
```

Comment:
Nit: this conflicts with the requirement in the load balancer spec that there must be an event that transitions the topology to `unknown`, whereas this leaves it optional. I suggest changing this to `MUST` as well.
The "replicaset-emit-topology-description-changed-before-close" test confirms this by explicitly checking for an `unknown` state on the topology description.